cpio/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+pkgname=cpio
```

Comment:
Add header

Reply:
@Alexpux What is the header? Is it okay if I insert something like `# Maintainer: Eon Jeong`?

Reply:
You'll need at least an email for contact info, but I think that you understand what @Alexpux is talking about. :)

Reply:
@Uni- I already made a pull request that fixes the issue as described above, but if you add yourself as the maintainer, I would prefer yours over mine.

Reply:
@ErichDonGubler @Adsun701 Thank you for all your kind comments.